### PR TITLE
Added missing changes.

### PR DESCRIPTION
**Pull request by Paebbels**
Added some missing changes.

### CODE REVIEW

File: .github/workflows/Pipeline.yml [hunk 1]
```diff
@@ -17,11 +17,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - {python: '3.6', icon: 🔴}   # until 23.12.2021
-          - {python: '3.7', icon: 🟠}  # until 27.06.2023
-          - {python: '3.8', icon: 🟡}  # until Oct. 2024
-          - {python: '3.9', icon: 🟢}  # until Oct. 2025
-          - {python: '3.10', icon: 🟢} # until Oct. 2026
+          - {python: "3.6",  icon: 🔴}   # until 23.12.2021
```
Comment (umarcor) [inline .github/workflows/Pipeline.yml:20]: There is an extra space before `#`.

File: doc/Dependency.rst [hunk 2]
```diff
@@ -26,12 +26,11 @@ pySVModel Package
...
++--------------------------------------------------------+-------------+------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------+
+| **Package**                                            | **Version** | **License**                                                                              | **Dependencies**                                                                                                                |
++========================================================+=============+==========================================================================================+=================================================================================================================================+
+| `pyTooling <https://github.com/pyTooling/pyTooling>`__ | ≥1.4.2      | `Apache License, 2.0 <https://github.com/pyTooling/pyTooling/blob/master/LICENSE.txt>`__ |                                                                                                                                 |
```
Comment (umarcor) [inline doc/Dependency.rst:32]: `github.com` is lowercase here.

File: doc/Dependency.rst [hunk 3]
```diff
@@ -46,11 +45,15 @@ sub-dependencies are not evaluated further.
 +-----------------------------------------------------------+-------------+----------------------------------------------------------------------------------------+----------------------+
 | **Package**                                               | **Version** | **License**                                                                            | **Dependencies**     |
 +===========================================================+=============+========================================================================================+======================+
-| `pytest <https://github.com/pytest-dev/pytest>`__         | ≥6.2.4      | `MIT <https://github.com/pytest-dev/pytest/blob/master/LICENSE>`__                     | *Not yet evaluated.* |
+| `pytest <https://GitHub.com/pytest-dev/pytest>`__         | ≥6.2.5      | `MIT <https://GitHub.com/pytest-dev/pytest/blob/master/LICENSE>`__                     | *Not yet evaluated.* |
```
Comment (umarcor) [inline doc/Dependency.rst:48]: It is camelcase here: `GitHub.com`.

File: doc/Dependency.rst [hunk 4]
```diff
@@ -66,7 +69,7 @@ evaluated further.
...
 | **Package**                                                                                     | **Version**  | **License**                                                                                              | **Dependencies**     |
 +=================================================================================================+==============+==========================================================================================================+======================+
-| `Sphinx <https://github.com/sphinx-doc/sphinx>`__                                               | ≥4.2.0       | `BSD 3-Clause <https://github.com/sphinx-doc/sphinx/blob/master/LICENSE>`__                              | *Not yet evaluated.* |
+| `Sphinx <https://github.com/sphinx-doc/sphinx>`__                                               | ≥4.3.0       | `BSD 3-Clause <https://github.com/sphinx-doc/sphinx/blob/master/LICENSE>`__                              | *Not yet evaluated.* |
```
Comment (umarcor) [inline doc/Dependency.rst:72]: It's lowercase again: `github.com`.

File: .github/workflows/Pipeline.yml (hunk 1 above)
Comment (Paebbels) [reply .github/workflows/Pipeline.yml:20]: Oh I got it. It's a problem in PyCharm. The font renders a red dot in different size, so I added one more space to align the comments ...

image

Comment (Paebbels) [reply .github/workflows/Pipeline.yml]: @umarcor fixed it. please review again.

**Review by umarcor — APPROVED** [review]

**Pull request merged**